templates/zerver/help/status-and-availability.md

```diff
@@ -65,26 +66,37 @@ There are four possible availabilities:
 * **Idle** (<span class="indicator orange"></span>): Zulip is open on
   your computer (either desktop or web), but you are not active.
 
-* **Offline** (<span class="indicator grey"></span>): Zulip is not
-  open on your computer.
+* **Offline** (<span class="indicator grey"></span>): Zulip is not open
+  on your computer, or you have turned on invisible mode.
 
-* **Unavailable** (<span class="indicator grey-line"></span>): You can
-  always manually set your availability to unavailable.
+You can see when someone offline was last recorded as active by hovering
+over their name in the left or right sidebar.
 
+### Invisible mode
 
-You can see when someone was last recorded as active by hovering over
-their name in the left or right sidebar (even if the user is marked as
-unavailable).
+Zulip supports the privacy option of never updating the availability
+information for your account. The result is that you will always
+appear to other users as **Offline**, regardless of your activity in
+Zulip.
+
+With this setting, your "Last active" time displayed to other users in
+the UI will be frozen as the approximate time you enabled this setting.
+
+!!! warn ""
+
+    **Note**: Because this setting works by making your availability
+    stop updating, you'll still appear to other users as active for a
+    few minutes after turning on invisible mode.
 
-### Set yourself as unavailable
+### Turn on invisible mode
```

Comment:
Let's make this "Toggle invisible mode", make the third instruction capture both, and drop the tip about using the same menu.

Reply:
Oh, sorry, I should have specified the text. I was just thinking:

> Click **Go invisible** or **Turn off invisible mode**.

Reply:
They'll only see one option, and it should be self-evident from the text what it does, so I don't think we need to explain anything more there.

Reply:
I think version A is probably best.